plaid/src/bin/plaid.rs

```diff
@@ -53,7 +53,7 @@ async fn main() -> Result<(), Box<dyn std::error::Error>> {
 
     info!("Configurating APIs for Modules");
     // Create the API that powers all the wrapped calls that modules can make
-    let api = Api::new(config.apis, log_sender.clone(), delayed_log_sender);
+    let api = Api::new(config.apis, log_sender.clone(), delayed_log_sender).await;
```

Comment:
Ah, this is a pretty major change. I think it's fine but have we considered any other places this could be an issue?

Reply:
Yeah I wanted to avoid this if possible but unfortunately has to happen. I can't think of any scenario where this would be ready before a rule ended up needing it. We could always block on this if we're concerned

Reply:
I think it's fine